test/product.e2e-spec.ts

```diff
@@ -304,6 +304,95 @@ describe('Product e2e', () => {
     );
   });
 
+  it('update DirectScriptureProduct with scriptureReferencesOverride errors', async () => {
+    const product = await createProduct(app, {
+      engagementId: engagement.id,
+    });
+
+    await expect(
+      app.graphql.query(
+        gql`
+          mutation updateProduct($input: UpdateProductInput!) {
+            updateProduct(input: $input) {
+              product {
+                ...product
+              }
+            }
+          }
+          ${fragments.product}
+        `,
+        {
+          input: {
+            product: {
+              id: product.id,
+              scriptureReferencesOverride: ScriptureRange.randomList(),
+            },
+          },
+        }
+      )
+    ).rejects.toThrowError();
```

Comment:
Can you specify the error message, type, or code here so that we assert exactly what we are trying to check for here?